examples/developer-tools/src/index.html

```diff
@@ -0,0 +1,23 @@
+<!doctype html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Blockly Developer Tools</title>
+  </head>
+  <body>
+    <div id="blockFactoryContainer">
```

Comment:
I'm no html expert, so I'm curious, is there a reason to wrap all of the html contents in a div instead of just applying the styling to the body?

Reply:
Eventually there will be other "main" divs other than just block factory, e.g. to switch between tabs to do toolbox stuff too.